src/librustc/metadata/creader.rs

```diff
@@ -698,7 +698,7 @@ pub fn import_codemap(local_codemap: &codemap::CodeMap,
             return false;
         }
 
-        for (&line1, &line2) in lines1.iter().zip(lines2.iter()) {
+        for (&line1, &line2) in lines1.iter().zip(&*lines2) {
```

Comment:
Similar to @steveklabnik's concern above, I'd stick to `.iter()` here as it seems more clear than `&*`